src/plugin.ts

```diff
@@ -0,0 +1,94 @@
+import {
+  IPolicyValidationPluginBeta1,
+  IPolicyValidationContextBeta1,
+  PolicyValidationPluginReportBeta1,
+} from 'aws-cdk-lib';
+import { Report, processReport } from './report';
+import { exec } from './utils';
+
+export interface CheckovValidatorProps {
+  /**
+     * List of checks to run
+     *
+     * @default - all checks are run
+     */
+  readonly check?: string[];
```

Comment:
the singular name is a bit confusing IMO - since it's a list I'd expecti to be plular, wdyt?

Reply:
I know what you mean, I kept it in sync with the naming in the `checkov` CLI. I'll tag @tsmithv11 for his opinion.